padertorch/contrib/examples/vae_mnist/mnist_vae_example.py

```diff
@@ -9,58 +9,62 @@
 """
 import os
 
-import padertorch
+import padertorch as pt
 import padertorch.train.optimizer as pt_opt
 
 import sacred
 
 import torch
 import torch.nn as nn
-import torchvision
-import torchvision.transforms as transforms
 
 from einops import rearrange
 
+from padertorch.contrib.je.data.transforms import Collate
```

Comment:
You should try not to be dependent on functions from contrib, better use pt.data.collate_fn

Reply:
I have used the function pt.data.utils.collate_fn. But i get a list of Tensors here, so that i can put them into my net, i had to edit them like this:  " torch.tensor([tensor.numpy() for tensor in inputs['image']])[:, None, :, :] "this was with Collate not necessary. Maybe you can help me to find a better solution.